### ISSUE TITLE
Away_mode in Alpha-Core-Preview

### ISSUE THREAD

**Issue by tmp-hallenser** [BenPru/luxtronik#127]
Hi,

in the latest Alpha-Core-Preview, the water_heater element looks as follows:

<img width="578" alt="Bildschirmfoto 2023-09-15 um 12 49 03" src="https://github.com/BenPru/luxtronik/assets/1869257/3fe99d54-2cdd-4cbe-bb18-bec0599c07af">

The bottom left selection is working, the bottom right selection seems to have no effect (i.e. operation state is not switched into 'off' in case 'away' is enabled)

**Comment by BenPru** [comment]
How do I get this tile?
What kind of one is that exactly?
Is this a standard Love Loce tile?

**Comment by tmp-hallenser** [comment]
yes, it's the standard tile. Just a the following line to a dashboard 
```
      - type: entities
        entities:
          - water_heater.luxtronik2_domestic_water
```
and click on the icon.

**Comment by BenPru** [comment]
I get this GUI:
![image](https://github.com/BenPru/luxtronik/assets/32298537/0509be37-c97e-4ccc-9015-b077e4ed2990)
![image](https://github.com/BenPru/luxtronik/assets/32298537/ae0f3998-d61e-438e-aebb-5be36d09902a)

**Comment by tmp-hallenser** [comment]
Is the switch "Away mode" at the bottom working?

**Comment by BenPru** [comment]
> Is the switch "Away mode" at the bottom working?

Yes.
Off=Off
Away=Holidays
Heat Pump=Auto
Electric=Sec. Heat Generator
Performance=Party
Works all fine for me.

**Comment by tmp-hallenser** [comment]
I'll wait for the next core preview and give it a new try (in the current one, it doesn't work). The "Away mode" switch doesn't have any effect.

**Comment by BenPru** [comment]
> I'll wait for the next core preview and give it a new try (in the current one, it doesn't work). The "Away mode" switch doesn't have any effect.

Please try [2023.9.21-Alpha-Core-Preview](https://github.com/BenPru/luxtronik/releases/tag/2023.9.21-Alpha-Core-Preview)

**Comment by tmp-hallenser** [comment]
No, still not working. (I'm on the latest docker image)

**Comment by BenPru** [comment]
@tmp-hallenser Please look in the log. Something with luxtronik?
Please provide a diagnostic file as attachment. See [Support tickets](https://github.com/BenPru/luxtronik/tree/main#support-tickets)
![image](https://github.com/BenPru/luxtronik/assets/32298537/9e2f23e3-e311-4a10-9db5-65171dd29d49)

**Comment by tmp-hallenser** [comment]
I'd wish I could download file, but it reads '500 Internal Server Error Server got itself in trouble" ... 

Can someone else reproduce the behavior or is it really just me?

**Comment by BenPru** [comment]
Please try the latest version.

**Comment by tmp-hallenser** [comment]
Changing the ``away_mode`` still does not change operation mode ...

The ``climate.luxtronik2_heating`` entity has also a bug from my perspective. It shows ``number.luxtronik2_heating_circuit_curve2_temperature_2`` as the target and no actual. Wouldn't it make more sense it to show ``number.luxtronik2_heating_target_correction``? Or show ``luxtronik2_flow_out_temperature_target`` (however, can't be set)

**Comment by BenPru** [comment]
> Changing the `away_mode` still does not change operation mode ...

I have no idea. Anyone else with this problem? I need more input or logs or diags...

> The `climate.luxtronik2_heating` entity ,,, shows `number.luxtronik2_heating_circuit_curve2_temperature_2` as the target and no actual.

What makes you think that about heating_circuit_curve2_temperature_2?
climate.luxtronik*_heating uses as actual what you have configured in the luxtronik integration options:
![image](https://github.com/BenPru/luxtronik/assets/32298537/f71f75d1-a993-4082-88e8-37d5170fe855)

Are you really sure you have a correct setup with a current version?

**Comment by rhammen** [comment]
For me, the Away mode works correctly.
Without additional logging, it is indeed very difficult to get more information.
I tried if I could understand why 'tmp-hallenser' got an error while downloading the diagnostics file.
And I also got an error when trying to download the diagnostics file. I created #163 for that.

@tmp-hallenser Could you try the following:
- restart HA
- try to set the away-mode in the DHW device
- go to System-->Logs, then click "Download Full Logs"
- post the downlloaded file in this thread.
Maybe we can get some insight in your problem using this approach.

**Comment by BenPru** [comment]
The diag download should work now:
Please try the latest prerelease: [2023.10.20-Alpha](https://github.com/BenPru/luxtronik/releases/tag/2023.10.20-Alpha)

**Comment by rhammen** [comment]
@tmp-hallenser 
Does release 2024.10.05-Beta now work OK for you?
Can this issue be closed?
If you still experience a problem, please clarify.
